src/types/redis_string.cc

```diff
@@ -297,6 +297,7 @@ rocksdb::Status String::SetRange(engine::Context &ctx, const std::string &user_k
     }
 
     Metadata metadata(kRedisString, false);
+    raw_value.clear();
```

Comment:
I think it'd be better to clear the raw_value inside `getRawValue` if the return status isn't ok. @torwig @PragmaTwice @mapleFU What do you think?

Reply:
@git-hulk Valid point. Since the value has expired, we should return "nothing" (empty) to the caller of `getRawValue`.

Reply:
Good to me. Actually refactor them to StatusOr can solve the problem, but I don't have much time to do that.